src/sentry/integrations/utils/code_mapping.py

```diff
@@ -276,16 +276,29 @@ def _generate_code_mapping_from_tree(
             for src_path in repo_tree.files
             if self._potential_match(src_path, frame_filename)
         ]
```

Comment:
```suggestion
        ]
        if len(matched_files) != 1
             return []
```

Reply:
This was just missing a `:` but it was added